src/app/home/home.component.scss

```diff
@@ -44,6 +49,17 @@ clr-row.selected{
     background-color: #f0f0f0;
 }
 
+.userlistIcon{  
+  color: #666666;  
+}
+.userlistIcon:hover{  
+  color: #3a3a3a;
+}
+
```

Comment:
Do we really want to deviate from the default colors that Clarity assigns to interactive elements like links and buttons? Do we already use non-standard colors elsewhere?